WORKING_GROUPS.md

```diff
@@ -14,7 +14,8 @@ think it would benefit from being done as an autonomous project.
 
 If the work defined in a Working Group charter is completed the Working
 Group should be dissolved and the responsibility for governance absorbed
-back in to the CTC.
+back in to the CTC. A Working Group can be revoked either through consensus 
+of the Working Group membership or normal CTC motion and vote.
```

Comment:
I think that's fine @Trott ... but then we should update the TSC's version too huh?